.github/workflows/build-package.yml

```diff
@@ -169,3 +168,49 @@ jobs:
 
       - name: Report deployed pacakge
         run: echo '### New release! 🚀`${{ steps.findPr.outputs.pr }}.${{ github.run_number }}` deployed at [Test Pypi](https://test.pypi.org/project/mpyl/)' >> $GITHUB_STEP_SUMMARY
+
+  Integration_Test:
```

Comment:
This test should be a tag run, otherwise it's testing the same thing as the mpyl test pipeline in jenkins.

Reply:
Also that is the flow that breaks often, since we don't (have means to) test it well.

Reply:
@koeves this is a prototype. I think we can easily add a tag build as well.

Reply:
I know I know, just saying we should add a tag flow :)

Reply:
https://github.com/Vandebron/mpyl/pull/233/commits/afd2f2e9e5e0a2f4d4715b2cb79219674caeb5cd